api/WebSocketError.json

```
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "WebSocketError": {
+      "__compat": {
+        "tags": [
+          "web-features:websocket-stream"
```

Suggesting for the first instance only:

```suggestion
          "web-features:web-socket-stream"
```